sdks/go/pkg/beam/core/graph/fn.go

```diff
@@ -167,6 +167,23 @@ const (
 	// TODO: ViewFn, etc.
 )
 
+var methodNames = []string{
```

Comment:
Consider using a map[string]struct{} instead instead of looping over a slice.

Alternatively, given that this is a combination of both doFnNames and sdfNames, consider  
var methodNames = append(append(nil, doFnNames...), sdfNames...)

which avoids duplicating the constant lists.

Reply:
Yeah I think I'll go with the set approach instead.